tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/lock/LocksInMainTest.java

```diff
@@ -257,7 +257,7 @@ public void testLockNegativeCases() {
                 18, 9);
     }
 
-    @Test(description = "Test for parallel run using locks")
+    @Test(description = "Test for parallel run using locks", enabled = false)
```

Comment:
Shall we create a high priority issue to track enabling this?

Reply:
Created the issue to track it:https://github.com/ballerina-platform/ballerina-lang/issues/22926